docs/web/contribute/environment.md

```diff
@@ -0,0 +1,90 @@
+# Setting up a Development Environment
+
+There are two primary ways to setup a development environment.
+
+- `nix`: fewer steps, isolated
+- `conda`: more steps, not isolated
+
+## Required Dependencies
+
+- [`git`](https://git-scm.com/)
+
+## Package Management
+
+!!! warning "You need at least one package manager"
+
+    At least one of `nix` or `conda` is required to contribute to ibis.
```

Comment:
Cannot one contribute using only poetry/pip?

Reply:
Probably, but this was already here and if I mention that, then I have to add another section. Happy to do that in a follow up.